components/FeatureHighlight/src/FontThemer/MDCFeatureHighlightFontThemer.h

```diff
@@ -31,7 +31,7 @@
  @param featureHighlightView A MDCFeatureHighlightView instance to apply a font scheme. Use a
      UIAppearance proxy to apply a font scheme to all instances of MDCFeatureHighlightView.
  */
-+ (void)applyFontScheme:(nonnull id<MDCFontScheme>)fontScheme
++ (void)applyFontScheme:(nonnull id<MDCFontScheming>)fontScheme
```

Comment:
This will be a breaking change - will this make the next release difficult?

Reply:
It is a breaking change.  The next release will be more difficult, however, the Font Scheme API is relatively new and there are 0 internal clients currently using it.